rabbitmq/README.md

```diff
@@ -55,6 +59,106 @@ To configure this check for an Agent running on a host:
 
 2. [Restart the Agent][8].
 
+##### Migrating from Management Plugin to Prometheus Plugin.
+
+If you were using this integration with the Rabbitmq Management Plugin and would like to switch to the Prometheus Plugin, this section is for you.
+
+You can switch the configuration by introducing a `prometheus_plugin` section in your instance configuration. As soon as you do so, any settings related to the Management Plugin are ignored. Below is an example of a minimal config sufficient to make the switch.
+
+```yaml
+instances:
+  - prometheus_plugin:
+      url: http://<HOST>:15692
+```
+
+This enables scraping of the [`/metrics` endpoint][20] on one rabbitmq node.
```

Comment:
Let's move this section under `Preparing RabbitMQ`, above the management plugin instructions.

```suggestion
##### RabbitMQ Prometheus Plugin
```
We can put the remap table in a new section at the bottom

Reply:
👍 I'll apply your suggestion then do the move locally